### PR TITLE
dact-1816 remove stop screen validation controller

### PR DESCRIPTION
**Pull request by AndreKramer7654**
JIRA https://companieshouse.atlassian.net/jira/software/c/projects/DACT/boards/492?selectedIssue=DACT-1816
Remove Stop screen validation controller.

### CODE REVIEW

File: routes.yaml [hunk 1]
```diff
@@ -2,8 +2,7 @@ app_name: officer-filing-api
 group: api    
 weight: 900
 routes:
-  1: ^/officer-filing
+  1: ^/officer-filing/healthcheck
```
Comment (jbestwickch) [inline routes.yaml:5]: Not quite clear on why this change is required?

Comment (AndreKramer7654) [reply routes.yaml:5]: Tightening up access. The only thing on the /officer-filing path is the healthcheck now.

**Review by jbestwickch — APPROVED** [review]

**Review by PhilipGlover — APPROVED** [review]
👍
